shared/lib-bie-kafka/src/main/java/gov/va/vro/model/biekafka/ContentionEvent.java

```diff
@@ -20,9 +20,9 @@ public String getTopicName() {
   }
 
   public static ContentionEvent mapTopicToEvent(String topic) {
-    // remove first word prefix from topic seperated by _
-    String noPrefixTopic = topic.substring(topic.indexOf("_") + 1);
-
+    // TODO(3/5/24): Add a better
```

Comment:
I do like the way this comment starts, but I'd prefer a little more clarification about what better something is coming.  You may already be planning to circle back with clarity on this, but what is your intent. If we are planning to leave this TODO in place, let's clarify further, or if you can address it now, please follow through with adding the better and remove the TODO.

Reply:
Hahaha thanks for calling this out, that TODO was out of date as soon as I wrote it because @chengjie8 supplied the regex below. I didn't plan on leaving the TODO in place post-merging.